test/AutoDiff/array.swift

```diff
@@ -289,6 +269,20 @@ ArrayAutoDiffTests.testWithLeakChecking("ArrayLiteralControlFlow") {
     let pb = pullback(at: Tracked<Float>(3), 4, in: { controlFlowGeneric($0, $1) })
     expectEqual((1, 1), pb(TrackedFloatArrayTan([1, 1])))
   }
+
+  do {
+    // Test nested array literal and control flow.
+    func controlFlowNestedLiteral(
+        _ x: Tracked<Float>, _ y: Tracked<Float>, _ bool: Bool = true
```

Comment:
2 extra spaces, blargh - I'll fix this later
```suggestion
      _ x: Tracked<Float>, _ y: Tracked<Float>, _ bool: Bool = true
```